botfront/imports/ui/components/nlu/models/NLUModel.jsx

```diff
@@ -263,7 +271,7 @@ class NLUModel extends React.Component {
         return (
             <div id='nlu-model'>
                 <Menu pointing secondary>
-                    <Menu.Item>{this.getHeader()}</Menu.Item>
+                    <Menu.Item style={{ paddingBottom: '0.3rem' }} header>{this.getHeader()}</Menu.Item>
```

Comment:
Don't add style in style node property, it's better to write it in a .less file because Meteor will automatically optimize code in those files.
You probably don't have to add a class, just write a css query specific enough

Reply:
Done